llvm/lib/Target/AVR/AVRISelDAGToDAG.cpp

```diff
@@ -514,7 +514,7 @@ bool AVRDAGToDAGISel::selectMultiplication(llvm::SDNode *N) {
   assert(Type == MVT::i8 && "unexpected value type");
 
   bool isSigned = N->getOpcode() == ISD::SMUL_LOHI;
-  unsigned MachineOp = isSigned ? AVR::MULSRdRr : AVR::MULRdRr;
```

Comment:
It would be better to keep the original name of `MULSRdRr`, most instructions are named in this tradition.

Reply:
Done